### PR TITLE
Add NAMD IDWS dataset

### PR DESCRIPTION
**Pull request by jhenin**
In support of PR https://github.com/alchemistry/alchemlyb/pull/135

@orbeckst please have a look. Thanks!

### CODE REVIEW

File: src/alchemtest/namd/access.py [hunk 1]
```diff
@@ -30,3 +30,25 @@ def load_tyr2ala():
...
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files by alchemical leg
+        - 'DESCR': the full description of the dataset
+
+    """
+
+    module_path = dirname(__file__)
+    data = {'forward': glob(join(module_path, 'idws/*.fepout.bz2'))}
```
Comment (orbeckst) [inline src/alchemtest/namd/access.py:47]: If it's only a single file then you can be explicit
```suggestion
    data = {'forward': [join(module_path, 'idws', 'quickidws.fepout.bz2']}
```

File: src/alchemtest/namd/idws/descr.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    :Number of Legs: 1 (forward mutation in water with IDWS sampling)
+    :Number of Windows: 11
+    :Length of Windows: 500 fs (each window interspersed with 100 fs equilibration)
+    :System Size: 1030 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: dummy mutation of ethane into ethane using dual topology
+                         hybrid molecule. Nonbonded interactions of perturbed
+                         atoms are scaled with their environment.
+    :Experimental Free Energy difference: N/A
```
Comment (orbeckst) [inline src/alchemtest/namd/idws/descr.rst:22]: I agree that "experimental" makes no sense. We don't really control the meta data in a rigorous fashion so I suggest you replace it with
```suggestion
    :Theoretical Free Energy difference: 0
```
which will be useful to anyone reading it. It's an exact theoretical number.

**Review by orbeckst — APPROVED** [review]
LGTM, minor suggestions which you can accept & add to the PR if you like. 

I also built the docs locally and everything renders nicely.

**Comment by jhenin** [comment]
Thanks for the suggestions @orbeckst ! Good to go as far as I'm concerned.

**Comment by orbeckst** [comment]
Thank you!

**Pull request merged**